packages/core/src/components/datetime/datetime.tsx

```diff
@@ -50,6 +51,9 @@ export class TdsDatetime {
   /** Label text for the component */
   @Prop() label: string = '';
 
+  /** Position of the label */
+  @Prop() labelPosition: 'inside' | 'outside' | 'no-label' = 'outside';
+
```

Comment:
Set "no-label" as default as it is a new prop so we do not surprise a user with new feature. Also other components tend to have "no-label" as fallback/default option.

```suggestion
  /** Position of the label */
  @Prop() labelPosition: 'inside' | 'outside' | 'no-label' = 'no-label';

```

Reply:
I addressed this here

> 2: I set the default labelPosition to outside (rather than no-label which is the default for other components such as text-field) to not cause any "breaking changes" for users currently using tegel, so that their label doesn't just disappear (with the current solution before this PR, there is no labelPosition prop so the label is always shown as long as the label prop contains some text

Considering that, do you still think no-label should be defaul?

Reply:
Yes, please do set it to "no-label". Thanks for double check ;)